conda_build/utils.py

```diff
@@ -1258,3 +1258,14 @@ def insert_variant_versions(metadata, env):
                 reqs.insert(i, ensure_valid_spec(' '.join((x.group(1),
                                                 metadata.config.variant.get(x.group(1))))))
     metadata.meta['requirements'][env] = reqs
+
+
+def patch_pyc_timestamp(filepath):
+    with open(filepath, 'rb') as pyc_file:
+        pyc_bytes = pyc_file.read()
+
+    new_timestamp = (DUMMY_TIMESTAMP + 1).to_bytes(4, byteorder='little')
```

Comment:
Hi @mandeep, please change byteorder='little' to use detection of endianness by checking the first 4 bytes. Even if all of the platforms we currently build for are little-endian, we shouldn't assume that's the case for everyone else.